scipy/spatial/transform/rotation.py

```diff
@@ -1461,6 +1463,43 @@ def magnitude(self):
         else:
             return angles
 
+    @classmethod
+    def create_group(cls, group, axis='Z'):
+        """Create a 3D rotation group.
+
+        Parameters
+        ----------
+        group: string
```

Comment:
Formatting `group : string` is more common and looks nicer in this context in my opinion.

Reply:
Fixed.